modules/odc-installing-helm-charts.adoc

```diff
@@ -5,26 +5,19 @@
 [id="odc-installing-helm-charts_{context}"]
 = Installing Helm charts
 
-You can use the *Developer* perspective to create Helm releases from the Helm charts provided in the *Developer Catalog*.
-
-.Prerequisites
-
-* You have logged in to the web console and have switched to the *Developer* perspective.
+You can use either the *Developer* perspective or the CLI to create Helm releases and see them in the Developer perspective of the web console.
 
 .Procedure
+To create Helm releases from the Helm charts provided in the *Developer Catalog*:
 
-. In the *Developer* perspective, navigate to the *Add* view and select the *From Catalog* option to see the *Developer Catalog*.
-. In the filters listed on the left, under *Type*, select the *Helm Charts* filter to see the available Helm charts.
+. In the *Developer* perspective, navigate to the *Add* view and select the *Helm Chart* option to see all the Helm Charts in the *Developer Catalog*.
+. Select a chart to see the description, README, and other details of the Helm chart and click *Install Helm Chart*.
 +
 image::odc_helm_chart_devcatalog.png[Helm charts in developer catalog]
 +
-. Select the Node.js chart to see the description of the chart and click *Install Helm Chart*.
 . In the *Install Helm Chart* page:
 .. Enter a unique name for the release in the *Release Name* field.
 .. Optionally, in the YAML editor, modify the YAML file.
-. Click *Install* to create a Helm release. The release is displayed in the *Topology* view.
-+
-image::odc_helm_release_topology.png[Helm release in topology]
-+
-. In the left panel of the *Developer* perspective, click *More* > *Helm* to see the list of all the Helm releases in the namespace.
-. Click on a listed Helm release to see the details and resources for that release.
+. Click *Install* to create a Helm release. The release is displayed in the *Topology* view. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
```

Comment:
When user clicks on the install button to install the Helm release, it will redirect him to the topology page. Can we modify the sentence like this?

Reply:
Fixed